File: Python/LongestIncreasingSubsequence.py
def longestSubsequence(nums):
    dp = [1] * len(nums)
    dp[0] = 1

    for i in range(1,len(nums)):
        for j in range(i):
            if nums[i] > nums[j]:
                dp[i] = max(dp[i], 1+dp[j])


    return max(dp)

File: Javascript/LongestIncreasingSubsequence.js
var longestSubsequence = function(nums){
    const dp = new Array(nums.length).fill(0)
    dp[0] = 1

    for(let i=1;i<nums.length;i++){
        for (let j=0;j<i;j++){
            if(nums[i]>nums[j]){
                dp[i] = Math.max(dp[i],1+dp[j])
            }
        }
    }

    return Math.max(...dp)
}